### DISCUSSION TITLE
Bureau hexagonable triple cannelure - tampon

### DISCUSSION THREAD

**Discussion by JoBalcony** [samuelhackwill/le-tiret-du-six#44] [Show and tell]
Hey guys,
after a little work with getting used to inkscape, here is a first impression of what the signature stamp on the cardboard desktops could look like.
The stamp is the same for each one of the desk's elements but by representing all three of them it allows to indicate witch one it is by checking the box.

![Capture du 2021-10-25 22-59-50](https://user-images.githubusercontent.com/25937530/138770467-654b441c-3563-4f81-9858-f737ab7a9241.png)

**Comment by samuelhackwill** [comment]
friggin sweet!

**Reply by JoBalcony** [reply]
Thanks,
will perhaps change typo and if I make the time.

I also think you closed the chrono/antichrono thread too fast.

**Reply by samuelhackwill** [reply]
I needed to make a decision for etienne to start working on the css/html.
do you think it's a bad decision? (you can still access closed issues)

**Comment by JoBalcony** [comment]
![tampon_bureau_2](https://user-images.githubusercontent.com/25937530/139741089-7427d8f6-cdfa-4e0c-bf74-539e71476f85.png)

nouveau tampon ?

**Comment by JoBalcony** [comment]
c'est quoi les dimensions des étiquettes ?

**Comment by samuelhackwill** [comment]
I'm not sure you want my opinion but i think the first version had better lettering (not a fan of the stretched text) & better overall composition. the labels are 62*100mm.
